src/mmw/js/src/modeling/tr55/quality/views.js

```diff
@@ -0,0 +1,92 @@
+"use strict";
+
+var Marionette = require('../../../../shim/backbone.marionette'),
+    coreModels = require('../../../core/models.js'),
```

Comment:
If we're only ever using `coreModels.DataCollection` in this file, the code below could be made a little DRYer by importing:

```javascript
DataCollection = require('../../../core/models.js').DataCollection,
```

and then just using `new DataCollection({})` instead of `new coreModels.DataCollection({})`